### PR TITLE
Updated ingress.yml snippet to use networking.k8s.io/v1

### LINKED ISSUE #901: Snippets: ingress.yaml should use apiVersion: networking.k8s.io/v1
Snippet [`ingress.yaml`](https://github.com/Azure/vscode-kubernetes-tools/blob/master/snippets/ingress.yaml) currently uses `apiVersion: extensions/v1beta1`, which is deprecated in v1.14+ (and will be removed in v1.22). It should therefore be replaced with `apiVersion: networking.k8s.io/v1`.

Proposed body for the snippet, in accordance with the [docs](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#ingressservicebackend-v1-networking-k8s-io): 
```
apiVersion: networking.k8s.io/v1
  kind: Ingress
  metadata:
    name: ${1:myingress}
    labels:
        name: ${1:myingress}
  spec:
    rules:
    - host: ${2:<Host>}
      http:
        paths:
        - pathType: Prefix
          path: "/"
          backend:
            service:
              name: ${3:<Service>}
              port: 
                number: ${4:<Port>}
```

### PR DESCRIPTION
**Pull request by Persistent13**
This resolves #901

### CODE REVIEW

File: snippets/ingress.yaml [hunk 1]
```diff
@@ -1,8 +1,4 @@
-name: ingress
-label: Kubernetes Ingress
-description: Kubernetes Ingress
-body: |2
```
Comment (bacongobbler) [inline snippets/ingress.yaml:4]: Can you add these back? It looks like you not only changed the apiVersion but you also changed the snippet.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

File: snippets/ingress.yaml (hunk 1 above)
Comment (Persistent13) [reply snippets/ingress.yaml:4]: Oops! Sorry about that

File: snippets/ingress.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 name: ingress
 label: Kubernetes Ingress
 description: Kubernetes Ingress
-body: |2 
-  apiVersion: extensions/v1beta1
+body: |2
+apiVersion: networking.k8s.io/v1
```
Comment (bacongobbler) [inline snippets/ingress.yaml:5]: needs to be indented 2 spaces to match.

**Review by bacongobbler — CHANGES_REQUESTED** [review]

**Comment by itowlson** [comment]
@bacongobbler is this okay to merge now?

**Review by bacongobbler — APPROVED** [review]

**Pull request merged**